Exercises_7_a_b_c.cpp

```diff
@@ -0,0 +1,98 @@
+
+#include "stdafx.h"
```

Comment:
stdafx.h is particular to windows (pre-compiled headers for speed), best not to include it for everyone.